quick_tour/metadata_filtering.ipynb

```diff
@@ -0,0 +1,2904 @@
+{
```

Comment:
<p>Just make it : Insert Vectors </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>